rcl/include/rcl/arguments.h

```diff
@@ -29,13 +29,13 @@ extern "C"
 {
 #endif
 
-struct rcl_arguments_impl_t;
+typedef struct rcl_arguments_impl_s rcl_arguments_impl_t;
 
 /// Hold output of parsing command line arguments.
-typedef struct rcl_arguments_t
+typedef struct rcl_arguments_s
 {
   /// Private implementation pointer.
-  struct rcl_arguments_impl_t * impl;
+  struct rcl_arguments_impl_s * impl;
```

Comment:
Given that we have all of the `typedef`s now, we could change all of these to:

```suggestion
  rcl_arguments_impl_t * impl;
```

But this is nitpicking at this point; I'm going to approve and leave it up to you whether you want to do that change.

Reply:
That's reasonable. See https://github.com/ros2/rcl/pull/932/commits/1d853818e1bba464aeb1361a9df37dc43614657a.